app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
+  def perform(keyword_id)
+    keyword = Keyword.includes(:source).find(keyword_id)
```

Comment:
It would be safer if we check the existence of the keyword for the background job.

```suggestion
    keyword = Keyword.includes(:source).find(keyword_id)
    return unless keyword

```